test/course_mapper_test.js

```diff
@@ -0,0 +1,47 @@
+const expect = require('chai').expect
+const CourseMapper = require('../src/mappers/course_mapper')
+
+describe('CourseMapper', function() {
+
+  let db = {
+    collection: function(collectionName) {
+      return {
+        find: function() {
+          return [{}, {}, {}]
+        },
+        insert: function(nome) {
+          return nome
+        }
+      }
+    }
+  }
+
+  it('return courses', function() {
+
+    let courseMapper = new CourseMapper(db)
```

Comment:
Poderia ser feito antes de todos os casos de teste (antes de todos os `it`). Talvez faça sentido o uso de um [hook](https://mochajs.org/#hooks) `beforeEach` aqui.